go.work

```diff
@@ -0,0 +1,6 @@
+go 1.19
+
+use (
+	.
```

Comment:
would it make sense to move these to a separate directory as well, rather than having them as an ancestor of the decryptionplugin?

Reply:
You mean making `tdh2` and `tdh2easy` separate modules?

Reply:
The latter